2.4/root/usr/share/container-scripts/httpd/common.sh

```diff
@@ -6,11 +6,57 @@ else
   HTTPCONF_LINENO=151
 fi
 
+gen_ssl_certs() {
+  local sslcert=$HTTPD_TLS_CERT_PATH/localhost.crt
+  local sslkey=$HTTPD_TLS_CERT_PATH/localhost.key
+  local fqdn=`hostname`
+
+  # A >59 char FQDN means "root@FQDN" exceeds 64-char max length for emailAddress
+  if [ "x${fqdn}" = "x" -o ${#fqdn} -gt 59 ]; then
+    fqdn=localhost.localdomain
+  fi
+
+  if [ -f ${sslcert} -o -f ${sslkey} ]; then
+    return 0
+  fi
+
+  if [ -f "/etc/fedora-release" ] || ( [ -f "/etc/redhat-release" ] && [ head "/etc/redhat-release" | grep -q "^Red Hat Enterprise Linux release 8" ] ); then
```

Comment:
Hm, the test failure on rhel8 looks like some issue in the code that checks the RHEL version:

https://gist.github.com/rhscl-bot/7a7544380ea58100d930b52622453d67#file-httpd-container-19-sh-L1322:
```
/usr/share/container-scripts/httpd//common.sh: line 23: [: missing `]'
grep: ]: No such file or directory
```